src/als/ui/dialogs.py

```diff
@@ -45,20 +50,28 @@ def __init__(self, parent=None):
 
         config_to_image_save_type_mapping[config.get_image_save_format()].setChecked(True)
 
-        self._show_missing_folders()
+        self._validate_all_pathes()
 
     @log
-    def _show_missing_folders(self):
+    def _validate_all_pathes(self):
         """
         Draw a red border around text fields containing a path to a missing folder
         """
+        warning_style = "border: 1px solid orange"
```

Comment:
Those are constants, please consider using CAPS and '_' prefix

Example : _MY_NICE_CONSTANT

Reply:
Done.